src/Mutate.tsx

```diff
@@ -122,8 +122,12 @@ class ContextlessMutate<TData, TError> extends React.Component<
 
     const requestPath =
       verb === "DELETE" && typeof body === "string"
-        ? url.resolve(base!, url.resolve(path, body))
-        : url.resolve(base!, path);
+        ? path.startsWith("/")
```

Comment:
See the other comment about nested ternaries. This also appears to be duplicated logic so maybe put it in a function that is unit tested?